integration/integration_test.go

```diff
@@ -583,7 +583,7 @@ func TestProfile(t *testing.T) {
 		{
 			name:           "Run profile gadget",
 			cmd:            fmt.Sprintf("$KUBECTL_GADGET profile cpu -n %s -p test-pod -K --timeout 15", ns),
-			expectedRegexp: `sh;\w+;\w+;\w+open`, // echo is builtin.
+			expectedRegexp: `sh;`, // echo is builtin.
```

Comment:
Why is it relevant that `echo` is a builtin? Opening the file /dev/null should still be done by the shell.

What is the output you see on arm?

Reply:
The comment regarding `echo` being a built-in is present since almost day 0 of this test, I wrote to explain why the regex contains `sh` and not `echo`.

Reply:
> What is the output you see on arm?

During the integration test, the output is the following:

```
sh;el0_sync;el0_sync_handler;el0_svc;do_el0_svc;el0_svc_common.constprop.0;invoke_syscall;__arm64_sys_openat
;do_sys_open;do_sys_openat2;do_filp_open;path_openat;path_init;nd_jump_root;set_root;set_root 1
        sh;el0_sync;el0_sync_handler;el0_svc;do_el0_svc;el0_svc_common.constprop.0;invoke_syscall;__arm64_sys_openat
;do_sys_open;do_sys_openat2;do_filp_open;path_openat;alloc_empty_file;percpu_counter_add_batch;percpu_counter_add_ba
tch 1
        sh;el0_sync;el0_sync_handler;el0_svc;do_el0_svc;el0_svc_common.constprop.0;invoke_syscall;__arm64_sys_write;
ksys_write;__fdget_pos;__fget_light;__fget_light 1
        sh;el0_sync;el0_sync_handler;el0_svc;do_el0_svc;el0_svc_common.constprop.0;el1_irq;gic_handle_irq;__handle_d
omain_irq;irq_exit;__softirqentry_text_start;rcu_core_si;rcu_core;rcu_do_batch;file_free_rcu;kmem_cache_free;__slab_
free;arch_local_irq_restore;arch_local_irq_restore 1
        sh;el0_sync;el0_sync_handler;el0_svc;do_el0_svc;el0_svc_common.constprop.0;invoke_syscall;__arm64_sys_openat;do_sys_open;do_sys_openat2;do_filp_open;path_openat;do_open.isra.0;may_open;inode_permission;inode_permission 1
        sh;el0_sync;el0_sync_handler;el0_svc;do_el0_svc;el0_svc_common.constprop.0;invoke_syscall;__arm64_sys_openat;do_sys_open;do_sys_openat2;do_filp_open;path_openat;alloc_empty_file;__alloc_file;security_file_alloc;kmem_cache_alloc;slab_post_alloc_hook;slab_post_alloc_hook 1
...
```